documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -3,77 +3,72 @@
 // deploying/deploying.adoc
 
 [id='ref-operator-cluster-feature-gates-{context}']
-= Enabling Strimzi feature gates
+= Feature gates
 
 [role="_abstract"]
-Strimzi operators use feature gates to enable or disable specific features and functions. 
-By enabling a feature gate, you alter the behavior of the corresponding operator, thereby introducing the feature to your Strimzi deployment.
+Strimzi operators use feature gates to enable or disable specific features and functions.
+Enabling a feature gate alters the behavior of the associated operator, introducing the corresponding feature to your Strimzi deployment.
 
-A feature gate might be enabled or disabled by default, depending on its level of maturity.
+The purpose of feature gates is to facilitate the trial and testing of a feature before it is fully adopted. 
+The preview state (enabled or disabled) of a feature gate may vary by default, depending on its maturity level.
```

Comment:
I would avoid calling it a preview as it is not always the purpose of the feature gate to preview something. Often it is also to gradually introduce some changes.

```suggestion
The state (enabled or disabled) of a feature gate may vary by default, depending on its maturity level.
```